### ISSUE TITLE
Vue 3 support?

### ISSUE THREAD

**Issue by dima-kov** [overtrue/vue-avatar-cropper#363]
Do you plan to add vue 3 support?

**Comment by overtrue** [comment]
Welcome to submit PR or wait for me to implement it in a few days. 😆

**Comment by dima-kov** [comment]
Ok, thank you very much

**Comment by turboteddy** [comment]
Any progress on this? Waiting for v3 support :)

**Comment by jonathan-ruiz** [comment]
I was about to try to update the code and send a PR, but using the source file and adding it as a component rather than using the minified file in /idst worked for me

**Comment by StevenMalaihollo** [comment]
Not working for me when importing it as a component. Still looking for a good Image Cropping package in Vue 3.

**Comment by MordiSacks** [comment]
I simply tried to copy the component into my project and it **worked** without any tweaking

**Comment by wduandy** [comment]
@overtrue hello! how are you? thanks for you contribution! are you still planning to give vue 3 support?

**Comment by ferretwithaberet** [comment]
> @overtrue hello! how are you? thanks for you contribution! are you still planning to give vue 3 support?

I wanted to try to use `vue-demi` to make this work with Vue 3. I did not have the time yet to get into it, but as far as I went, it does not seem like `vue-demi` is the right solution. You are free to contribute with ideas of how to use the same code base for both vue 2 and vue 3, otherwise we will need to make a new repo/branch which will hold the code for the vue 3 version of this component, which would need features to be added to each branch individually.
